RecoHI/HiTracking/python/HITrackingRegionProducer_cfi.py

```diff
@@ -88,3 +88,15 @@
 HiTrackingRegionFactoryFromSTAMuonsBlock.MuonTrackingRegionBuilder.vertexCollection = cms.InputTag("hiSelectedVertex")
```

Comment:
@echapon please confirm on the last question from @makortel as well.
Thank you.